scrapinghub/client.py

```diff
@@ -774,7 +780,17 @@ class Logs(_Proxy):
         [{
             'level': 20,
             'message': '[scrapy.core.engine] Closing spider (finished)',
-            'time': 1482233733976},
+            'time': 1482233733976,
+        }]
+
+    - retrive logs with a given log level and filter by a word
+
+        >>> filters = [("message", "contains", ["logger"])]
+        >>> list(job.logs.iter(level='WARNING', filter=filters))
```

Comment:
I know I asked that before but maybe we should have a `entity.list()` method instead of `list(entity.iter())` pattern?